app/services/DateService.scala

```diff
@@ -33,4 +33,6 @@ class DateService @Inject()(appConfig: config.AppConfig) {
     }
   }
 
+  val isPreCovidDeadline: Boolean = if (now().isAfter(LocalDate.of(2020,6,30))) true else false
```

Comment:
Slightly confusing naming here - variable is called `isPreCovidDeadline` but the logic here will return `true` if the deadline has passed? Also, you don't need the if statement as `isAfter` will already return you a boolean.